docker-compose.infra_only.yml

```diff
@@ -34,7 +34,7 @@ services:
             CELERY_BROKER_URL: ${CELERY_BROKER_URL}
             TASK_COLUMNS: "name,uuid,state,args,kwargs,result,received,started,runtime,worker,retries,revoked,exception,eta"
             FLOWER_PORT: 5555
-            FLOWER_UNAUTHENTICATED_API: true
+            FLOWER_UNAUTHENTICATED_API: "true"

```

Comment:
AFAIK environment variables are always strings. So even if yaml has some boolean data type when docker compose reads the config it can only set this environment block as string values. So I doubt that this is necessary. Or am I missing something?

Reply:
Indeed - YAML rarely needs quotes, unless you need to force it as a string. 

From my experience, reading a Docker Compose file with quotes... seems just weird. 
That's far from an official spec - I know ;-)

The `FLOWER_UNAUTHENTICATED_API = true` comes directly from the Flower docs; not the safest of practices but not accessible or running in PROD anyway.

The issue with double quotes was because of the environment vars or `.env` file parsing in Python. 
We never put quotes around string variables in them ([example](https://github.com/buildingSMART/validate/blob/main/.env)).